### ISSUE TITLE
prefer-arrow/prefer-arrow-function: prefer arrow to allow top level named functions as default

### ISSUE THREAD

**Issue by kbrilla** [typescript-eslint/tslint-to-eslint-config#838]
### 🚀 Feature Request

Currently migration will just output notice that top level named functions are disallowed by the plugin, but default tslint behaviour is to allow them. I think default migration should enable allowStandaloneDeclarations configuration parameter of the prefer-arrow/prefer-arrow-functions rule to match tslint rule behaviour. 

#### Existing Behavior

top level function declarations allowed as default

#### Change Proposal

```json
  "prefer-arrow/prefer-arrow-functions": [
          "error",
          {
            "allowStandaloneDeclarations": true
          }
        ]
``` 
as default migration

**Comment by KingDarBoja** [comment]
Did I miss something? AFAIK, the [TSLint rule](https://palantir.github.io/tslint/rules/only-arrow-functions/) states:

> Disallows traditional (non-arrow) function expressions.

And provides an option to allow named functions (`allow-named-functions`), which I believe is for top-level named functions.

**Comment by criskrzysiu** [comment]
I'm pretty sure the TsLint rule didn't report such functions, as I had no errors like this while linting. After conversion EsLint is reporting them so they have to be a difference in their behaviors.

**Comment by KingDarBoja** [comment]
i am still not 100% sure about the correct behaviour, tried to understand the differences following this [issue](https://github.com/TristonJ/eslint-plugin-prefer-arrow/issues/17) at the plugin repository but based on this [one](https://github.com/TristonJ/eslint-plugin-prefer-arrow/issues/16), it seems you are right, so feel free to go ahead and make a PR 🚀

**Comment by JoshuaKGoldberg** [comment]
Just following up, I tried out `npx tslint index.ts` with:

```json
// tslint.json
{
  "rules": {
    "only-arrow-functions": true
  }
}
```

```index.ts
// index.ts
function myFunc() {

const myFunc = function () {}
```

...and TSLint did complain by default:

```plaintext
$ npx tslint index.ts
index.ts:1:1
ERROR: 1:1   only-arrow-functions  non-arrow functions are forbidden
ERROR: 3:16  only-arrow-functions  non-arrow functions are forbidden
```
